src/01.UI-Elements/AddButton.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import AddIcon from '@material-ui/icons/Add'
+
+AddButton.propTypes = {
+  onClick: PropTypes.func.isRequired,
+}
+
+export default function AddButton({ onClick }) {
+  return (
+    <AddButtonStyled onClick={onClick} data-testid="add-button">
+      <AddIconStyled />
+      <ButtenTextStyled>Listeneintrag</ButtenTextStyled>
+    </AddButtonStyled>
+  )
+}
+
+const AddButtonStyled = styled.button`
+  margin-left: 4px;
+  margin-bottom: 20px;
+  border: none;
+  background: none;
+  display: flex;
+  align-items: center;
+`
+const AddIconStyled = styled(AddIcon)`
+  margin-right: 20px;
+  color: var(--mid-gray);
+`
+
+const ButtenTextStyled = styled.span`
+  color: var(--light-gray);
+`
```

Comment:
You really wanted that whole thing to be a clickable Button? Including the text "Listeneintrag" ? I expected "Listeneintrag" to be smth. like a Placeholder where i can input my "item text", but instead it instantly created an empty Item.